src/main/kotlin/com/team4099/robot2023/RobotContainer.kt

```diff
@@ -62,6 +88,13 @@ object RobotContainer {
     //
     // ControlBoard.advanceAndClimb.whileActiveOnce(AdvanceClimberCommand().andThen(RunClimbCommand()))
     //        ControlBoard.climbWithoutAdvance.whileActiveOnce(RunClimbCommand())
+
+    ControlBoard.extendIntake.whileTrue(IntakeExtendCommand(intake))
```

Comment:
they were mainly just for testing but the final command could be built off them. since intake command would be pretty simple idk if we'd want to do this but you could use IntakeExtendCommand in execute and then IntakeRetractCommand in interrupt of another command to create the functionality ur talking abt